FlowersStore/Scenes/FlowersList/FlowersListPresenter.swift

```diff
@@ -20,16 +20,26 @@ final class FlowersListPresenter {
     private weak var output: FlowersListPresenterOutput?
```

Comment:
<!--
  0 failure: 
  1 warning:  Property 'output'...
  
  
  DangerID: danger-id-Danger;
  File: FlowersStore/Scenes/FlowersList/FlowersListPresenter.swift;
  Line: 20;
-->

- :warning: Property 'output' is assigned, but never used